src/App.tsx

```diff
@@ -1,34 +1,103 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api/api';
+import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
+import { Selected } from './types/Selected';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/96409563/f5f2b3ea-a73e-45bb-bcd1-3c2651bf2e70)